ui/pages/confirmations/components/simulation-details/amount-pill.tsx

```diff
@@ -78,6 +87,7 @@ export const AmountPill: React.FC<{
         position="bottom"
         title={tooltipParts.join(' ')}
         wrapperStyle={{ minWidth: 0 }}
+        theme="word-break-all"
```

Comment:
Unfortunately `Tooltip` ignores `style` property hence this is the only proper solution that I find.